README.md

```diff
@@ -85,7 +143,3 @@ the [components](#components) section for further details.
 ## Glossary of Terms
 
 See the [glossary of terms](Glossary.md) related to Kata Containers.
```

Comment:
I think we could link straight to https://github.com/kata-containers/kata-containers/wiki/Glossary here while we're at it.